packages/cardano-services/src/Asset/DbSyncAssetProvider.ts

```diff
@@ -0,0 +1,88 @@
+import { Asset, AssetProvider, Cardano, ProviderError, ProviderFailure } from '@cardano-sdk/core';
+import { DbSyncProvider } from '../DbSyncProvider';
+import { Logger } from 'ts-log';
+import { NetworkInfoBuilder } from '../NetworkInfo/DbSyncNetworkInfoProvider/NetworkInfoBuilder';
+import { NftMetadataService, TokenMetadataService } from './types';
+import { Pool } from 'pg';
+
+/**
+ * Dependencies that are need to create DbSyncAssetProvider
+ */
+export type DbSyncAssetProviderDependencies = {
+  /**
+   * The PGPool to db-sync
+   */
+  db: Pool;
+
+  /**
+   *
+   * The logger object
+   */
+  logger: Logger;
+
+  /**
+   * The NftMetadataService to retrieve Asset.NftMetadata .
+   */
+  ntfMetadataService: NftMetadataService;
+
+  /**
+   * The TokenMetadataService toretrieve Asset.TokenMetadata .
+   */
+  tokenMetadataService: TokenMetadataService;
+};
+
+type AssetExtraData = Parameters<AssetProvider['getAsset']>[1];
+
+/**
+ * AssetProvider implementation using NftMetadataService, TokenMetadataService
+ * and cardano-db-sync database as sources
+ */
+export class DbSyncAssetProvider extends DbSyncProvider implements AssetProvider {
+  #builder: NetworkInfoBuilder;
+  #deps: DbSyncAssetProviderDependencies;
```

Comment:
```suggestion
  #dependencies: DbSyncAssetProviderDependencies;
```
For consistency with other classes such as https://github.com/input-output-hk/cardano-js-sdk/blob/32886498fe2b308e2e8ea45507c806dd8fd9eab8/packages/rabbitmq/src/rabbitmqTxSubmitProvider.ts#L46

Reply:
Resolved in 0c7f451ab